packages/amplify_authenticator/lib/src/services/amplify_auth_service.dart

```diff
@@ -167,6 +170,21 @@ class AmplifyAuthService implements AuthService {
     }
   }
 
+  @override
+  Future<bool> get isValidSession async {
```

Comment:
It probably doesn't make sense for this to be a getter. I generally would not expect a getter to have any side effects or throw exceptions.
```suggestion
  Future<bool> isValidSession() async {
```

Reply:
Yeah good call.